exercises/simple-linked-list/src/example.c

```diff
@@ -0,0 +1,127 @@
+#include "simple_linked_list.h"
+
+/*
+Initialize new node
+*/ 
+    node * new_node(int data) 
+{
+   
node * startp = (node *) malloc(sizeof(node));
```

Comment:
... on these lines also in the example.c, the example.h and the test_simple_linked_list.c
I'm not sure why the CI hasn't caught this.

Reply:
Also on this line - it is considered bad practice to cast the returned value of `malloc()`